cluster/charts/rook-ceph-cluster/templates/role.yaml

```diff
@@ -82,7 +82,7 @@ rules:
     verbs: ["get", "list", "delete" ]
   - apiGroups: [""]
     resources: ["persistentvolumeclaims"]
-    verbs: ["delete"]
+    verbs: ["get", "update", "delete"]
```

Comment:
Isn't `get` sufficient?

Reply:
`update` might also be needed because we [remove](https://github.com/rook/rook/blob/0d8fd9d8a47799fbb2607fded7bab757fee2fd6a/pkg/daemon/ceph/osd/remove.go#L119) the `ceph.rook.io/DeviceSetPVCId` label from the `pvc` if its decided to be preserved.